packages/CodeQuality/tests/Rector/Foreach_/SimplifyForeachToCoalescingRector/config.yml

```diff
@@ -0,0 +1,2 @@
+services:
+    Rector\CodeQuality\Rector\Foreach_\SimplifyForeachToCoalescingRector: ~
```

Comment:
This should also be added to `code-quality.yml` level

Reply:
I'll handle it with autoregistration of whole package :+1: